lib/src/main/java/com/github/kevinsawicki/http/HttpRequest.java

```diff
@@ -3178,6 +3178,19 @@ public HttpRequest form(final Map<?, ?> values, final String charset)
   }
 
   /**
+   * Write the json string
+   *
+   * @param jsonString
+   * @return this request
+   * @throws HttpRequestException
+   */
+  public HttpRequest json(final String jsonString) {
```

Comment:
This should probably be `final CharSequence` to be consistent with the signature of the other helpers.